### PR TITLE
Add registry-k8s-io sync scripts

### PR DESCRIPTION
**Pull request by BobyMCbobs**
These are the scripts that I have built and optimized for manual syncing k8s.gcr.io GCS bucket and the registry.k8s.io S3 buckets

- `initial-copy-to-s3.sh` copies from GCS to S3 (_us-east-2_)
- `initial-copy-between-s3.sh` copies from _us-east-2_ to the other buckets and uses tmate to run in parallel

The scripts are intended to be run by any IAM user with ability to assume the _registry.k8s.io_s3writer_ role in the _registry.k8s.io_ account, such as the _registry.k8s.io-ci_ user in the _k8s-infra-accounts_ account.

### CODE REVIEW

File: registry.k8s.io/hack/initial-copy-between-s3.sh
Comment (jaypipes) [inline registry.k8s.io/hack/initial-copy-between-s3.sh:47]: looks familiar! ;)

File: registry.k8s.io/hack/initial-copy-between-s3.sh
Comment (jaypipes) [inline registry.k8s.io/hack/initial-copy-between-s3.sh:74]: Interesting use of `tmux` ... nothing wrong with this per-se, but might want to mention in a comment at the top of this file that `tmate` is required to run the script.

File: registry.k8s.io/hack/initial-copy-between-s3.sh
Comment (jaypipes) [inline registry.k8s.io/hack/initial-copy-between-s3.sh:36]: here and throughout...

little nit: you know that `$REGION` will be defined already so no reason to use the defaulting syntax of `${REGION:-}`. You can just use `$REGION`

File: registry.k8s.io/hack/initial-copy-between-s3.sh
Comment (BobyMCbobs) [reply registry.k8s.io/hack/initial-copy-between-s3.sh:47]: It's a nifty few lines! Works well :smile:

File: registry.k8s.io/hack/initial-copy-between-s3.sh
Comment (BobyMCbobs) [reply registry.k8s.io/hack/initial-copy-between-s3.sh]: It's definitely all about manual local syncing. I've added a note in 7b1e676e1

File: registry.k8s.io/hack/initial-copy-between-s3.sh
Comment (BobyMCbobs) [reply registry.k8s.io/hack/initial-copy-between-s3.sh:39]: I write it like `${REGION:-}` instead of `$REGION` since sometimes `shellcheck` likes to have a fallback value, which is allowed to be empty. Something around variable declaration maybe?

**Review by jaypipes — APPROVED** [review]
ok dokey.

**Comment by ameukam** [comment]
/approve
/hold

Remove the hold when you're ready.

**Comment by BobyMCbobs** [comment]
Looking for an lgtm once again

/unhold

cc @ameukam @jaypipes

**Comment by dims** [comment]
/approve
/hold

Remove the hold when you're ready.

**Comment by BobyMCbobs** [comment]
Thank you @ameukam!

**Pull request merged**
